cocoa/vendor/bugsnag-cocoa/Source/BugsnagHandledState.m

```diff
@@ -116,8 +120,9 @@ + (NSString *)stringFromSeverityReason:(SeverityReasonType)severityReason {
             return kPromiseRejection;
         case UserSpecifiedSeverity:
             return kUserSpecifiedSeverity;
+        case LogMessage:
+            return kLogGenerated;
         case UnhandledException:
-        default:
             return kUnhandledException;
```

Comment:
Should we use `NSParameterAssert` for the default case here?

Reply:
If there's no default case, there's a build warning if any cases are missing. The change here was to specify every case so its not needed.